src/Toolkit/Toolkit.WPF/UI/Controls/TimeSlider/TimeSlider.Theme.xaml

```diff
@@ -161,19 +161,6 @@
                                                         Background="Transparent"
                                                         BorderBrush="Transparent" 
                                                         BorderThickness="0">
-                                                    <Grid Margin="1" Background="Transparent">
-                                                        <Border x:Name="BackgroundAnimation" Opacity="0" Background="#FF448DCA"/>
-                                                        <Rectangle x:Name="BackgroundGradient">
-                                                            <Rectangle.Fill>
-                                                                <LinearGradientBrush EndPoint=".7,1" StartPoint=".7,0">
-                                                                    <GradientStop Color="#FFFFFFFF" Offset="0"/>
-                                                                    <GradientStop Color="#F9FFFFFF" Offset="0.375"/>
-                                                                    <GradientStop Color="#E5FFFFFF" Offset="0.625"/>
-                                                                    <GradientStop Color="#C6FFFFFF" Offset="1"/>
-                                                                </LinearGradientBrush>
-                                                            </Rectangle.Fill>
-                                                        </Rectangle>
-                                                    </Grid>
```

Comment:
Could you explain why this part was removed?

Reply:
This specific part was causing an unexpected white background to be drawn behind the play button:

![image](https://user-images.githubusercontent.com/29742178/153104737-cb5fd8a5-af91-46f2-bacb-813ff44768c4.png)

It is the first checkbox in https://github.com/Esri/arcgis-toolkit-dotnet/issues/193